packages/victory-core/src/victory-util/textsize.js

```diff
@@ -240,6 +240,27 @@ const _approximateTextHeightInternal = (text, style) => {
   }, 0);
 };
 
+// Stubbable implementation.
+export const _approximateTextSizeInternal = {
```

Comment:
@boygirl Do you have thoughts about this stubbable implementation that @ryan-roemer added? Normally I'm not a huge fan of doing hacky stuff like this for the sake of testing, but I also don't want to lose the benefit of the existing tests. 

The other option we have (which probably isn't much better) is to do some old-fashioned dependency injection and accept the text size as an optional prop.

Reply:
@beccanelson I agree that the stubbable implementation isn't great, but I think it's the best option for the moment. Let's go with that so you don't have to keep spinning your wheels on this.

Reply:
I totally agree with y’all. 😛 If it was more than just one, we’d probably want to look for more first class solutions.